conans/client/tools/win.py

```diff
@@ -216,33 +275,18 @@ def vcvars_command(settings, arch=None, compiler_version=None, force=False):
             else:
                 _global_output.warn(message)
     else:
-        env_var = "vs%s0comntools" % compiler_version
-
-        if env_var == 'vs150comntools':
-            vs_path = os.getenv(env_var)
-            if not vs_path:  # Try to locate with vswhere
-                vs_root = vs_installation_path("15")
-                if vs_root:
-                    vs_path = os.path.join(vs_root, "Common7", "Tools")
-                else:
-                    raise ConanException("VS2017 '%s' variable not defined, "
-                                         "and vswhere didn't find it" % env_var)
-            if not os.path.isdir(vs_path):
-                _global_output.warn('VS variable %s points to the non-existing path "%s",'
-                    'please check that you have set it correctly' % (env_var, vs_path))
-            vcvars_path = os.path.join(vs_path, "../../VC/Auxiliary/Build/vcvarsall.bat")
+        vs_path = vs_installation_path(str(compiler_version))
+
+        if not vs_path or not os.path.isdir(vs_path):
+            _global_output.warn("VS non-existing installation")
+        else:
+            vcvars_path = ""
+            if int(compiler_version) > 14:
+                vcvars_path = os.path.join(vs_path, "VC/Auxiliary/Build/vcvarsall.bat")
+            else:
+                vcvars_path = os.path.join(vs_path, "VC/vcvarsall.bat")
             command = ('set "VSCMD_START_DIR=%%CD%%" && '
```

Comment:
Is this command valid for previous versions of VS? Does VSCMD_START_DIR affects?

Reply:
I just left this command as it was before this PR. Havent checked if it works

Reply:
Nop, it only applied to VS15 before this PR. Now applies to all VS versions, it seems.